src/main/scala/temple/generate/CRUD.scala

```diff
@@ -1,13 +1,13 @@
 package temple.generate
 
-sealed trait CRUD
+abstract class CRUD(val verb: String)
 
 object CRUD {
-  case object ReadAll extends CRUD
-  case object Create  extends CRUD
-  case object Read    extends CRUD
-  case object Update  extends CRUD
-  case object Delete  extends CRUD
+  case object ReadAll extends CRUD("List")
+  case object Create  extends CRUD("Create")
+  case object Read    extends CRUD("Read")
+  case object Update  extends CRUD("Update")
+  case object Delete  extends CRUD("Delete")
```

Comment:
Pedantic thought - we've done this elsewhere by keeping the base type as a sealed trait and then having a toString defined on each of the implementations - which do we like better? @xsanda ?

Reply:
Personally went for this method as `.verb`gives a little more semantic meaning than `.toString`, which could produce a debug description or something else unexpected

Reply:
This raises they question of why we have the difference between List and ReadAll. If we merge them then toString should be fair game

Reply:
![](https://media.giphy.com/media/UBdhsDANl8YQ8/giphy.gif)

Think changing that is a little out of scope for this PR